docs/community/glossary.md

```diff
@@ -0,0 +1,33 @@
+# OpenSauced Glossary
+Welcome to the glossary, where we define concepts used on our platform to help you get the most out of OpenSauced.
+
+## PR Velocity
+The PR Velocity measures the percentage and amount of days that pull requests that are merged. It is a great way to track the progress of your open source contributions.
```

Comment:
Let's add a space between the title and paragraph here.
Besides that it is the [best practice](https://www.markdownguide.org/basic-syntax/#heading-best-practices), we will do this going forward for consistency.

```suggestion
## PR Velocity

The PR Velocity measures the percentage and amount of days that pull requests that are merged. It is a great way to track the progress of your open source contributions.
```